app/views/layouts/styleguide.html.erb

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tahi</title>
+    <%= stylesheet_link_tag "application", media: "all" %>
+    <%= javascript_include_tag "application" %>
+
+    <%= csrf_meta_tags %>
+    <%= render "shared/bugsnag_javascript" %>
+
+    <style media="screen">
+    *[element-name] {
+      position: relative;
+    }
+    .add-author-form {
+      min-width: 900px;
+    }
+    .baskerville * {
+      font-family: Baskerville;
+    }
+    .buttons div.button--green,
+    .buttons div.button--grey,
+    .buttons div.button--blue {
+      margin-top: 10px;
+    }
+    .cabin * {
+      font-family: Cabin;
+    }
+    #card-overlays > a {
+      display: block
+    }
+    #cards-tab .overlay-container {
+      max-width: 940px;
+      min-width: 900px;
+      margin: 0;
+      padding: 0;
+    }
+    #cards-tab ul.select2-choices {
+      display: none;
+    }
+    .color-bar {
+      height: 75px;
+      text-align: center;
+      color: white;
+      padding: 28px 0px 0px 0px;
+      width: 200px;
+    }
+    .column-title-update-buttons {
+      display: none !important;
+    }
+    .constrain {
+      max-width: 940px;
+      margin: auto;
+    }
+    .content {
+      padding-left: 50px;
+      padding-right: 50px;
+    }
+    .glyphicon.large {
+      font-size: 28px;
+    }
+    .glyphicon.small {
+      font-size: 18px;
+    }
+    #header {
+      margin: 20px auto;
+    }
+    .navigation-toggle {
+      display: none !important;
+    }
+    p.text {
+      max-width: 500px;
+      padding-bottom: 25px;
+    }
+    .styleguide-example pre,
+    .styleguide-example .styleguide-description,
+    .styleguide-example h3,
+    .styleguide-example .styleguide-element .styleguide-modifier-name {
+      display: none;
+    }
+    .row {
+      padding-bottom: 50px;
+    }
+    #tabs {
+      margin-bottom: 20px;
+    }
+  </style>
```

Comment:
Consider pulling this style sheet into its own file, though it's not critical. We can do this later if the need arises.